code/upaya/Gemfile

```diff
@@ -2,9 +2,9 @@ source 'https://rubygems.org'
 ruby '2.3.0'
 
 gem 'rails', '~> 4.2.6'
-
 gem 'attr_encrypted'
 gem 'autoprefixer-rails', '~> 5.2'
+gem 'bullet'
```

Comment:
Should we add the `rails_12factor` gem, per these instructions? https://docs.cloud.gov/apps/rails/